ramls/po_line.raml

```diff
@@ -8,8 +8,8 @@ documentation:
     content: <b>This module implements the CRUD interface</b>
 
 types:
-    po_line: !include acq-models/mod-orders-storage/schemas/po_line.json
-    po_line_collection: !include acq-models/mod-orders-storage/schemas/po_line_collection.json
+    po-line: !include acq-models/mod-orders-storage/schemas/po_line.json
```

Comment:
I think the .raml files also have to follow the - convention. I did not find it anywhere on the naming conventions guide but all the raml-utils atleast seem to follow that convention

Reply:
All .raml files renamed to use '-' instead of '_'